packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -186,7 +186,6 @@ class VerticalCaretMovementRun extends BidirectionalIterator<TextPosition> {
     return true;
   }
 
-  @override
```

Comment:
This method now needs a doc comment since it is no longer inheriting the docs from the super class. Adding one should make the checks happy.